entities.py

```diff
@@ -198,26 +234,32 @@ def luck(self):
     def luck(self, val):
         self._generic_setter('_luck', val, 1)
 
-    @abstractmethod
-    def _compute_max_hp(self): pass
+    def _compute_max_hp(self):
+        # 25% STR, 75% CON, 100 BASE
+        hp_str = self.strength * self.level * 0.25
+        hp_con = self.constitution * self.level * 0.25
+        self.max_hp = hp_str + hp_con + 100
```

Comment:
Lots of arbitrary formulas in here. Maybe we shouldn't be implementing these until we start balancing the game. Goes for all compute methods.

Reply:
I am aware, I just put those in so that it works